pulp_smash/pulp3/utils.py

```diff
@@ -196,13 +196,16 @@ def inner(repo, version_href=None):
             return {}
 
         client = api.Client(config.get_config(), api.page_handler)
-        return client.get(version_href)[summary_field]
+        to_return = client.get(version_href)['content_summary'][summary_field]
+        for key in to_return:
+            to_return[key].pop('href')  # Figure out how to assert on changing hrefs in response
```

Comment:
@bmbouter I don't get why we need to remove the href from response here, can you elaborate a little more why this is being popped out?

Reply:
Well it really shouldn't be but it's the best I can offer before I go away for 2 weeks.

So the response formats look like this:  
![screenshot from 2019-02-25 17-54-43](https://user-images.githubusercontent.com/180319/53651598-f8a3af00-3c14-11e9-9b9a-2583c916dde8.png)

The `href` field changes depending on how many test have run before it so it can't easily be added to the fixture data. What do you think is the right way to resolve this issue in pulp smash?

Reply:
Previously Pulp had a bunch of different fields on a repository named

* content_summary
* content_added_summary
* content_removed_summary
* content_hrefs
* content_added_hrefs
* content_removed_hrefs

These are all being merged together to look like this

```
"content_summary: {
    'added': {
        'file.file': {
            'count': 5,
            'href': '/pulp/api/v3/content/file/files/?repository_version_added=/pulp/api/v3/repositories/1/versions/1/'
        }
    },
    'present': {
        'file.file': {
            'count': 5,
            'href': '/pulp/api/v3/content/file/files/?repository_version=/pulp/api/v3/repositories/1/versions/1/'
        }
    },
    'removed': {}
}
```

Since the pulp_smash API is going to stay the same (for now at least), it's a hack to make it look the same as it did before.

Reply:
Recapping our convo, I'm going to update this function to fully preserve the previous data layout and file a pulp-smash issue to handle creating a new util function with the new interface so users can port over.

Reply:
That issue is here:  https://github.com/PulpQE/pulp-smash/pull/1175